File: sdkwork-app-sdk-go/types/wallet_history_query_form.go
package types

// 钱包交易流水查询请求
type WalletHistoryQueryForm struct {
	TransactionType string `json:"transactionType"`
	Status string `json:"status"`
	StartDate string `json:"startDate"`
	EndDate string `json:"endDate"`
	PageNum int `json:"pageNum"`
	PageSize int `json:"pageSize"`
	SortField string `json:"sortField"`
	SortDirection string `json:"sortDirection"`
	AccountType string `json:"accountType"`
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/wallet_history_query_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class WalletHistoryQueryForm:
    """钱包交易流水查询请求"""
    transaction_type: str = None
    status: str = None
    start_date: str = None
    end_date: str = None
    page_num: int = None
    page_size: int = None
    sort_field: str = None
    sort_direction: str = None
    account_type: str
